aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -467,6 +455,22 @@ When hosting a non-ASP.NET Core sub-app underneath an ASP.NET Core app, explicit
 
 ::: moniker-end
 
+Static asset links within the sub-app should use tilde-slash (`~/`) notation. Tilde-slash notation triggers a [Tag Helper](xref:mvc/views/tag-helpers/intro) to prepend the sub-app's pathbase to the rendered relative link. For a sub-app at `/subapp_path`, an image linked with `src="~/image.png"` is rendered as `src="/subapp_path/image.png"`. The root app's Static File Middleware doesn't process the static file request. The request is processed by the sub-app's Static File Middleware.
+
+If a static asset's `src` attribute is set to a relative URL (for example, `src="/image.png"`), the link is rendered without the sub-app's pathbase. The root app's Static File Middleware attempts to serve the asset from the root app's [webroot](xref:fundamentals/index#web-root-webroot), which results in a *404 - Not Found* response unless the static asset is available from the root app.
```

Comment:
`/image.png` could more specifically be called an "absolute path" as opposed to a "relative path". "relative URL" is ambiguous in this case.